modules/mauth-test-utils/src/main/java/com/mdsol/mauth/test/utils/model/AuthenticationOnly.java

```diff
@@ -0,0 +1,27 @@
+package com.mdsol.mauth.test.utils.model;
+
+public class AuthenticationOnly implements TestCase {
+  String name;
+  CaseType type;
+  AuthenticationHeader authHeader;
+  UnsignedRequest unsignedRequest;
+
+  public AuthenticationOnly(String name) {
+    this.name = name;
+    this.type = CaseType.AUTHENTICATION_ONLY;
+  }
```

Comment:
Can you make the constructor take all the require parameters? (Much like a case class in Scala).

This makes it easier to refactor when you remove / add new fields.

Reply:
updated as suggested in https://github.com/mdsol/mauth-jvm-clients/pull/133/commits/a33ca9d7f45ba653dacdc14bbcef7d90d4c2a853